frontend/components/ModelForm.tsx

```diff
@@ -60,6 +67,7 @@ type ModelFormProps = {
   empty?: ReactElement | string
   fields: any
   tableFields?: TableField[]
+  filterOptions?: filterOption[]
```

Comment:
This should be title case: `FilterOption` instead of `filterOption`, just a code style guideline.